lib/taskana-simplehistory-provider/pom.xml

```diff
@@ -181,7 +181,7 @@
 		<dependency>
 			<groupId>pro.taskana</groupId>
 			<artifactId>taskana-core</artifactId>
-			<version>1.0.4</version>
+			<version>1.0.5-SNAPSHOT</version>
```

Comment:
Do you really need to change this dependency?

Reply:
I tried to avoid it, but the BaseQuery changed.
So the Syntax for the Query is slightly different to the other Query Mappers

Reply:
ok